pkg/gui/keybindings.go

```diff
@@ -543,6 +543,13 @@ func (self *Gui) GetInitialKeybindings() ([]*types.Binding, []*gocui.ViewMouseBi
 			Handler:     self.handleToggleSelectHunk,
 			Description: self.c.Tr.ToggleSelectHunk,
 		},
+		{
+			ViewName:    "main",
+			Contexts:    []string{string(context.MAIN_PATCH_BUILDING_CONTEXT_KEY), string(context.MAIN_STAGING_CONTEXT_KEY)},
```

Comment:
I don't think this works with `context.MAIN_PATCH_BUILDING_CONTEXT_KEY`. I assume the expected result there would be that we would modify the patch we were building instead of immediately applying it?